### DISCUSSION TITLE
Scripted Install

### DISCUSSION THREAD

**Discussion by stevefxp1** [pfelk/pfelk#363] [Q&A]
I have been fiddling with the scripted install and have a couple of questions:

1) The version is 20.3. Is this the latest and greatest, as I see references to 20.10
2) Does the scripted install configure ELK to startup automatically at reboot? It seems Kibana does not restart.
3) You show in the prereqs the setup of remote logging, but you also mention this again under the firewall configuration, after installation. One has pfELK as the description and the other says to name it Firewall. Any difference here?

Thanks,
Steve

**Comment by a3ilson** [comment]
@stevefxp1 

1. The latest script installer is version is [21.12](https://raw.githubusercontent.com/pfelk/pfelk/main/etc/pfelk/scripts/pfelk-installer.sh)
2. The script does not set Elastic, Logstash or Kibana to start on boot. This was considered but to keep things slim and reduce variables for errors it was not added.
  * To add simply do the following: 
```
sudo /bin/systemctl enable elasticsearch.service
sudo /bin/systemctl enable logstash.service
sudo /bin/systemctl enable kibana.service
```
3. No - just that remote logging is configured
